### PR TITLE
Exception homework

### PR DESCRIPTION
**Pull request by Jengero**

### CODE REVIEW

File: CSharpException/CSharpException/ReadDoubleFromConsole.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpException
+{
+    internal static class ReadDoubleFromConsole
```
Comment (MariaEroshevich) [inline CSharpException/CSharpException/ReadDoubleFromConsole.cs]: better to name class like ConsoleReader

File: CSharpException/CSharpException/ReadDoubleFromConsole.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            double value;
+            bool check = true;
+            while (check != double.TryParse(enteredValue, out value))
+            {
+                Console.WriteLine("Wrong value. Try again");
+                enteredValue = Console.ReadLine();
+            }
+            return value;
+        }
+
```
Comment (MariaEroshevich) [inline CSharpException/CSharpException/ReadDoubleFromConsole.cs]: remove spaces

File: CSharpException/CSharpException/Program.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿using CSharpException;
+double sideOne = Triangle.EnterSide();
+double sideTwo = Triangle.EnterSide();
+double sideThree = Triangle.EnterSide();
+
+Triangle triangleToExeptions = new(sideOne, sideTwo, sideThree);
+
+try
+{
+    if (sideOne > sideTwo + sideThree) { throw new SideValueException(sideOne, sideTwo, sideThree); }
```
Comment (MariaEroshevich) [inline CSharpException/CSharpException/Program.cs]: Move this verifications to separate method in Tringle class. Then call this method in try catch construction here.

File: CSharpException/CSharpException/Program.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    else if (sideTwo > sideOne + sideThree) { throw new SideValueException(sideTwo, sideOne, sideThree); }
+    else if (sideThree > sideOne + sideTwo) { throw new SideValueException(sideThree, sideOne, sideTwo); }
+}
+catch (SideValueException ex)
+{
+    Console.WriteLine(ex.Message);
+    return;
```
Comment (MariaEroshevich) [inline CSharpException/CSharpException/Program.cs]: remove return.

File: CSharpException/CSharpException/Triangle.cs [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        public double ThirdSide
+        {
+            get => _thirdSide;
+            internal set 
+            { 
+                _thirdSide = value; 
+            }
+        }
+
+        private double _firstSide;
```
Comment (MariaEroshevich) [inline CSharpException/CSharpException/Triangle.cs]: Fields should go first, then properties. And add spaces between properties.

**Review by MariaEroshevich — DISMISSED** [review]

File: CSharpException/CSharpException/ReadDoubleFromConsole.cs (hunk 1 above)
Comment (dnnyhlwdplc) [reply CSharpException/CSharpException/ReadDoubleFromConsole.cs]: Why do you have unused class here? I see that you have console reader already :)

File: CSharpLinq/CSharpLinq/Program.cs [hunk 6]
```diff
@@ -0,0 +1,99 @@
+﻿using CSharpOOP2;
+using System.Diagnostics;
+#region Task_1. List of numbers divisible by three
+//List<int> numbersToDivide = new() { 32, 50, 40, 77, 36, 93, 57, 4, 37, 7 };
```
Comment (dnnyhlwdplc) [inline CSharpLinq/CSharpLinq/Program.cs]: please be careful about changes in PR :)
I see LINQ homework here :)

**Review by dnnyhlwdplc — APPROVED** [review]

**Pull request merged**